src/devtools/devtools.tsx

```diff
@@ -467,6 +483,9 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
         {...panelProps}
       >
         <style
+          scoped
+          nonce={styleNonce}
+          data-testid="react-query-devtools-panel-style"
```

Comment:
we can just find the `style` tag without needing a test-id I guess, just by doing:

```
const { container } = render(<Page />)
const button = container.querySelector('style')
```